Add documentation for the JDBC adapter (#155)
scalar-labs/scalardb: docs/getting-started-with-scalardb.md

}
```

Note that you need to use `JdbcDatabaseConfig` when you use JDBC databases as follows.

```java

>     Injector injector = Guice.createInjector(new StorageModule(new JdbcDatabaseConfig(props)));

Reviewer: It's not a question to the doc, but do we still need to pass `JdbcDatabaseConfig` for StorageModule?
I think we updated to create `JdbcDatabaseConfig` in the module  for transaction.

}
```

Similar to the case of the storage service, when you use JDBC databases, you need to use `JdbcDatabaseConfig` as follows.

```java

>     Injector injector = Guice.createInjector(new TransactionModule(new JdbcDatabaseConfig(props)));

Reviewer: ```suggestion
    Injector injector = Guice.createInjector(new TransactionModule(new DatabaseConfig(props)));
```
I think it now works with this, doesn't it?

$ ../../gradlew run --args="-mode transaction -action pay -amount 100 -to merchant1 -from user1"
```

> ## Using JDBC transaction

Reviewer: ```suggestion
## Use JDBC transaction
```

> When you use a JDBC database, you can choose the transaction manager type you use.

Reviewer: ```suggestion
When you use a JDBC database as a backend database, you can optionally use the native transaction manager of a JDBC database instead of the default `ConsensusCommit` transaction manager.
```

> There are two transaction manager types, `consensuscommit` and `jdbc`.

Reviewer: ```suggestion
```

> `consensuscommit` is the default transaction manager type, which is also used in the other adapters (Cassandra, DynamoDB, and Cosmos DB).

Reviewer: ```suggestion
```

> `jdbc` is a transaction manager type that can be used only in JDBC databases, and it uses the transaction implementation of the JDBC database you use.

Reviewer: ```suggestion
```

> To use the `jdbc` transaction manager, you need to set the following property in **scalardb.properties**.

Reviewer: ```suggestion
To use the native transaction manager, you need to set `jdbc` to a transaction manager type in **scalardb.properties** as follows.
```

```
scalar.db.jdbc.transaction_manager.type=jdbc
```

You don't need to set a key `transaction` to `true` in Scalar DB scheme for the `jdbc` transaction manager.
So you can use the same scheme file as **emoney-storage.json**.

```json
{
  "emoney.account": {
    "transaction": false,
    "partition-key": [
      "id"
    ],
    "clustering-key": [],
    "columns": {
      "id": "TEXT",
      "balance": "INT"
    }
  }
}
```

> You can run the scalar schema standalone loader to load the schema as follows.

Reviewer: The following lines might be not needed?
